src/store/helpers.js

```diff
@@ -21,6 +21,6 @@ export const offlineConfig = {
             return retries === action.meta.offline.maxRetry;
```

Comment:
While you're fixing the default return value, do you mind changing this to a `>=`? Using `==` when checking the maximum always gets me a bit worried